### PR TITLE
Modify build_gh_pages.sh in order to create zipped maps for pipeline …

### PR DESCRIPTION
**Pull request by jannikr**
# Create zipped maps in pipeline build

Issue: #2538

We would like to test this modification with a patch release.

### CODE REVIEW

**Review by Hall-Ma — APPROVED** [review]

**Comment by MW-Friedrich** [comment]
If @ce-bo is alright with this, we could merge and release #2913 as a patch? 🤔

File: script/build_gh_pages.sh [hunk 1]
```diff
@@ -22,12 +22,16 @@ $CCSH sonarimport -nc -o codecharta_sonar.cc.json https://sonarcloud.io maibornw
 $CCSH modify --set-root root/maibornwolff-gmbh_codecharta_visualization -o codecharta_sonar_mod.cc.json codecharta_sonar.cc.json
 
 $CCSH merge -o ../visualization/app/codecharta.cc.json codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json -nc
+# Zipped map for pipeline build
+$CCSH merge -o ../visualization/app/codecharta.cc.json codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json
```
Comment (ce-bo) [inline script/build_gh_pages.sh:26]:
```suggestion
$CCSH merge -o ../visualization/app/codecharta.cc.json.gz codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json
```

**Comment by Hall-Ma** [comment]
> If @ce-bo is alright with this, we could merge and release #2913 as a patch to test this? 🤔

Yesterday Cedrik and I discussed this topic. We will release it as a patch.

File: script/build_gh_pages.sh [hunk 2]
```diff
@@ -22,12 +22,16 @@ $CCSH sonarimport -nc -o codecharta_sonar.cc.json https://sonarcloud.io maibornw
...
 
 # Map for analysis
-$CCSH sonarimport -nc -o codecharta_sonar_analysis.cc.json https://sonarcloud.io maibornwolff-gmbh_codecharta_analysis 
+$CCSH sonarimport -nc -o codecharta_sonar_analysis.cc.json https://sonarcloud.io maibornwolff-gmbh_codecharta_analysis
 $CCSH modify --set-root root/maibornwolff-gmbh_codecharta_analysis -o codecharta_sonar_mod.cc.json codecharta_sonar_analysis.cc.json
 $CCSH modify --set-root root/analysis -o codecharta_git_mod.cc.json codecharta_git.cc.json
 $CCSH merge -o ../visualization/app/codecharta_analysis.cc.json codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json -nc
+# Zipped map for pipeline build
+$CCSH merge -o ../visualization/app/codecharta_analysis.cc.json codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json
```
Comment (ce-bo) [inline script/build_gh_pages.sh:34]:
```suggestion
$CCSH merge -o ../visualization/app/codecharta_analysis.cc.json.gz codecharta_sonar_mod.cc.json codecharta_git_mod.cc.json
```

**Review by ce-bo — COMMENTED** [review]
Did zippedMap.cc.json.gz was added intentionally?
Let's add a changelog entry, so that the users know the reason for the upcoming patch release.

File: script/build_gh_pages.sh (hunk 2 above)
Comment (Hall-Ma) [reply script/build_gh_pages.sh:34]: I tried your suggestion and we don't need it. The merge filter puts `.gz` at the end automatically.

**Comment by jannikr** [comment]
> Did zippedMap.cc.json.gz was added intentionally? Let's add a changelog entry, so that the users know the reason for the upcoming patch release.

Yes, we did this for testing purposes, as a second approach to read zipped files.

File: script/build_gh_pages.sh (hunk 1 above)
Comment (Hall-Ma) [reply script/build_gh_pages.sh:26]: I tried your suggestion and we don't need it. The merge filter puts .gz at the end automatically.

**Pull request merged**